src/queryExecutionContext/SqlQuerySpec.ts

```diff
@@ -14,18 +14,18 @@
  * ```
  */
 export interface SqlQuerySpec {
-  /** The text of the SQL query */
-  query: string;
-  /** The parameters you provide in the query */
-  parameters?: SqlParameter[];
+    /** The text of the SQL query */
```

Comment:
This was automatic from the EditorConfig settings, but I can try and undo it if prefered